src/docserv/bih.py

```diff
@@ -0,0 +1,388 @@
+import json
+import time
+import subprocess
+import shlex
+import datetime
+import os
+import sys
+import threading
+import queue
+import socket
+import configparser
+import logging
+import signal
+import tempfile
+import string
+import random
+import hashlib
+from http.server import HTTPServer, BaseHTTPRequestHandler
+from socketserver import ThreadingMixIn
+from xml.etree import ElementTree, cElementTree
+from email.mime.text import MIMEText
+
+from docserv.deliverable import Deliverable
+from docserv.repolock import RepoLock
+from docserv.functions import resource_to_filename, mail
```

Comment:
It may be a nice addition: you could use relative imports. The only thing you need to change is to remove `docserv`.

Reply:
I tried, but then it fails in develop mode of setuptools.

Reply:
Never mind. Just one of my crazy ideas. :laughing: